Formula/pyqt5.rb

```diff
@@ -15,8 +16,8 @@ class Pyqt5 < Formula
 
   deprecated_option "enable-debug" => "with-debug"
 
-  depends_on :python3 => :recommended
-  depends_on :python => :optional
+  depends_on :python => :recommended
+  depends_on :python3 => :optional
```

Comment:
I remembered why they were the way round they were: otherwise the bottles will always require installing Homebrew's Python.